examples/View.elm

```diff
@@ -1,79 +1,165 @@
+        , Html.td [] [ Html.text (Maybe.withDefault "[empty]" description) ]
+        , Html.td [] [ Html.text (formatLastModified last_modified currentTime) ]
+        , Html.td []
             [ iconBtn "edit" (EditRecord id)
-            , text " "
+            , Html.text " "
             , iconBtn "trash" (DeleteRecord id)
             ]
         ]
 
 
-recordsList : List Record -> Time -> Html Msg
-recordsList records currentTime =
-    table [ class "table" ]
-        [ thead []
-            [ tr []
-                [ th [] [ text "id" ]
-                , th [] [ text "title" ]
-                , th [] [ text "description" ]
-                , th [] [ text "last_modified" ]
-                , th [] []
-                ]
+recordHeaders : Sort -> List (Html.Html Msg)
+recordHeaders sort =
+    let
+        ( icon, sortColumn ) =
+            case sort of
+                Asc column ->
+                    ( "glyphicon glyphicon-sort-by-attributes", column )
+
+                Desc column ->
+                    ( "glyphicon glyphicon-sort-by-attributes-alt", column )
+
+        headings =
+            [ "id", "title", "description", "last_modified" ]
+
+        headingWithSortIcon : String -> String -> String -> Html.Html Msg
+        headingWithSortIcon icon sortColumn heading =
+            let
+                content =
+                    if heading == sortColumn then
+                        [ Html.i [ Html.Attributes.class icon ] []
+                        , Html.text <| " " ++ heading
+                        ]
+                    else
+                        [ Html.text heading ]
+            in
+                Html.th [ Html.Events.onClick (SortByColumn heading) ] content
+    in
+        List.map
+            (headingWithSortIcon icon sortColumn)
+            headings
+
+
+recordsList : List Record -> Time -> Sort -> Html.Html Msg
+recordsList records currentTime sort =
+    Html.table [ Html.Attributes.class "table" ]
+        [ Html.thead []
+            [ Html.tr []
+                (recordHeaders sort)
             ]
-        , tbody [] (List.map (recordRow currentTime) records)
+        , Html.tbody [] (List.map (recordRow currentTime) records)
         ]
 
 
-errorNotif : Maybe String -> Html Msg
+errorNotif : Maybe String -> Html.Html Msg
 errorNotif error =
     case error of
         Nothing ->
-            text ""
+            Html.text ""
 
         Just message ->
-            div [ class "alert alert-danger" ] [ text ("Error: " ++ message) ]
+            Html.div [ Html.Attributes.class "alert alert-danger" ]
+                [ Html.text ("Error: " ++ message) ]
 
 
-sortedRecords : Records -> List Record
-sortedRecords records =
-    Dict.values records
-        |> List.sortBy .last_modified
-        |> List.reverse
+view : Model -> Html.Html Msg
+view { error, records, formData, currentTime, sort } =
+    Html.div [ Html.Attributes.class "container" ]
+        [ Html.h1 [] [ Html.text "Kinto Elm :-)" ]
```

Comment:
May be a good time to update the title with something like *elm-kinto demo* or smthg